doc/source/deployment/setup-deployer.rst

```diff
@@ -67,6 +68,11 @@ Deployer. Set up your workspace with the following steps:
   export SOCOK8S_ENVNAME=socok8s
   export SOCOK8S_WORKSPACE_BASEDIR=~/socok8s-workspace
 
+Prepare deployer to use virtualenv.
+
+..code-block:: console
+
+  export SOCOK8S_USE_VIRTUALENV=True
```

Comment:
I do not have the whole picture, but this should not be needed at the end. If user has installed from the package, all should work out of the box (and https://github.com/SUSE-Cloud/socok8s/pull/426 should help again)

Reply:
Agreed with @jsuchome .